src/components/UIShell/SideNav.js

```diff
@@ -79,7 +79,7 @@ export default class SideNav extends React.Component {
     });
 
     return (
-      <aside className={className}>
+      <aside aria-label="side navigation" className={className}>
```

Comment:
Should this be translatable?

Reply:
Oh, yeah probably! I'll investigate how to make that happen.

Reply:
Should this use what's in `accessibilityLabel`?